ghost/i18n/locales/sl/portal.json

```
@@ -15,6 +15,8 @@
     "Account": "Račun",
     "Account details updated successfully": "Podatki o računu so bili uspešno posodobljeni",
     "Account settings": "Nastavitve računa",
+    "Add a personal note": "",
+    "After a free trial ends, you will be charged the regular price for the tier you've chosen. You can always cancel before then.": "Po koncu brezplačnega preizkusa se vam zaračuna redna cena za izbrano stopnjo. Pred tem jo lahko vedno prekličete.",
```

Biome (1.9.4)

[error] The key After a free trial ends, you will be charged the regular price for the tier you've chosen. You can always cancel before then. was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)